plugins/transfer_pokemon/__init__.py

```diff
@@ -31,17 +32,41 @@ def log(text, color="black"):
 
     if bot.config.cp or bot.config.pokemon_potential:
         ignore_list = bot.config.ign_init_trans.split(',')
-        log(
-            "Transferring {} Pokemon with CP NOT above {} and IV NOT above {} and excluding {}.".format(
-                ("caught" if isinstance(pokemon, Pokemon) else "all"),
-                bot.config.cp,
-                bot.config.pokemon_potential,
-                ignore_list
+        if release_rules:
+            log(
+                "Transferring {} Pokemon according to rules and excluding [{}].".format(
+                    ("caught" if isinstance(pokemon, Pokemon) else "all"),
+                    ', '.join(ignore_list)
+                )
+            )
+        else:
+            log(
+                "Transferring {} Pokemon with CP NOT above {} and IV NOT above {} and excluding [{}].".format(
+                    ("caught" if isinstance(pokemon, Pokemon) else "all"),
+                    bot.config.cp,
+                    bot.config.pokemon_potential,
+                    ', '.join(ignore_list)
+                )
             )
-        )
 
         groups = list(indexed_pokemon.keys())
         for group in groups:
+            # Load rules for this group.
+            pokemon_name = bot.pokemon_list[group - 1]["Name"]
+            try:
+                pokemon_rules = release_rules[pokemon_name]
+                if 'release_below_cp' not in pokemon_rules:
+                    pokemon_rules['release_below_cp'] = bot.config.cp
+                if 'release_below_iv' not in pokemon_rules:
+                    pokemon_rules['release_below_iv'] = bot.config.pokemon_potential
+                if 'logic' not in pokemon_rules:
+                    pokemon_rules['logic'] = 'and'
+            except KeyError:
+                pokemon_rules = {
+                    'release_below_cp': bot.config.cp,
+                    'release_below_iv': bot.config.pokemon_potential,
+                    'logic': 'and',
+                }
```

Comment:
Can probably be done better by using `release_rules.get(pokemon_name, default)` where `default` is a default configuration.